api/android/sample/src/main/java/org/nnsuite/nnstreamer/sample/MainActivity.java

```diff
@@ -262,6 +262,9 @@ private void runSingle() {
             TensorsInfo outInfo = single.getOutputInfo();
             printTensorsInfo(outInfo);
 
+            /* set timeout (1 second) */
+            single.setTimeout(1000);
```

Comment:
Default timeout in C-API was 3 sec.

Reply:
This is sample code to set 1 second timeout.